doc/040_backup.rst

```diff
@@ -529,6 +529,7 @@ environment variables. The following lists these environment variables:
     AWS_ACCESS_KEY_ID                   Amazon S3 access key ID
     AWS_SECRET_ACCESS_KEY               Amazon S3 secret access key
     AWS_DEFAULT_REGION                  Amazon S3 default region
+    AWS_PROFILE                         Amazon credentials profile (can be used instead of the keys and region)
```

Comment:
Can you please do the following:

- Remove "of the" in your description for `AWS_PROFILE`.
- Add also the `AWS_SHARED_CREDENTIALS_FILE` one.